ecosystem/indexer-grpc/indexer-grpc-parser/README.md

```diff
@@ -17,9 +17,21 @@ Indexer GRPC parser is to indexer data processor that leverages the indexer grpc
         processor_name: default_processor
         postgres_connection_string: postgresql://postgres:@localhost:5432/postgres_v2
         indexer_grpc_data_service_addresss: 127.0.0.1:50051
+        indexer_grpc_keep_alive_interval_in_secs: 60
+        indexer_grpc_keep_alive_timeout_in_secs: 10
         auth_token: AUTH_TOKEN
     ```
 
+#### Config Explanation
+
+* `processor_name`: purpose of this processor; also used for monitoring purpose.
+* `postgres_connection_string`: PostgresQL DB connection string
+* `indexer_grpc_data_service_addresss`: Data service non-TLS endpoint address.
+* `indexer_grpc_keep_alive_interval_in_secs`: client-side HTTP2 KEEPALIVE check interval.
```

Comment:
actually can you add a bit better description here? When I just googled HTTP2 KEEPALIVE I didn't really find any useful results. I think what this is really referring to is using HTTP2 PING frames for health checking, the argument just happened to be name `http2_keepalive` in tonic (btw they name it without an underscore).
What actually gave me a better understanding what this is, is looking at the tonics for this flag.
Anyways, I'd suggest updating the description, potentially renaming the flag and also adding a link to the flag docs in tonic.

Reply:
good catch; i wasn't aware of their naming is inconsistent 🤦 .

done